### ISSUE TITLE
Hyper is Slow

### ISSUE THREAD

**Issue by gsklee** [zeit/hyper#1576]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Sierra 10.12.3
- **Hyper.app version**: 1.2.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: (pristine)
- **Relevant information from devtools**: N/A
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue

I typed a simple command of `$ nvm ls-remote` into both Hyper and iTerm2, hit enter in Hyper first then quickly switched to iTerm2 and enter. Turned out iTerm2 was way faster than Hyper in spewing out the result.

Screencast:
https://sendvid.com/fniaojdu

**Comment by rauchg** [comment]
@gsklee, we've merged some important improvements onto `master`, so I'll have you retry this demo once `1.3` is out please 👌

**Comment by gsklee** [comment]
@rauchg Cool, looking forward to it 😸

**Comment by albinekb** [comment]
soooo @gsklee faster? 😏

**Comment by CodeTheory** [comment]
Closing this due to inactivity. Please let us know if you still see performance issues on either stable or canary releases ❤️

**Comment by donaldpipowitch** [comment]
Hi! I just switched to Win 10 (1709) coming from Mac and use Hyper.js with my WSL. I don't know why, but it seems to slow down _a lot_ every now and then and becomes unresponsive. I tried the last stable version and the last canary version. It isn't slow the whole time, but it seems to slow down "the more happens": open one or two tabs, run `ll` or `git log`. Don't using tabs and maybe just use some simple commands seem to run well. Sometimes even the whole text disappears for several seconds until it appears again. Any ideas? :/
